platform_plugin_communications/api/serializers.py

```diff
@@ -0,0 +1,23 @@
+"""
+Serializers for the platform_plugin_communications app.
+"""
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+User = get_user_model()
+
+
+class UserSearchSerializer(serializers.ModelSerializer):
+    """
+    Serializer for User model.
+    """
+
+    name = serializers.CharField(source="profile.name", read_only=True)
```

Comment:
Can we send the name as in the Open edX APIs? I think it's inside the profile dict. Can we check how it's done?

Reply:
It's stored in the profile.name field: 
![image](https://github.com/eduNEXT/platform-plugin-communications/assets/33465240/4b1ff664-ffdf-4ec6-a351-88f664b93cf4)

Reply:
how is it returned by the users API? Like this?
```
{
"username": "admin",
"profile": {
  "name": "Administrator"
 }
}
```

Reply:
It uses the `name`:

![image](https://github.com/eduNEXT/platform-plugin-communications/assets/33465240/d277f03d-9146-42aa-a958-854a9c45197a)